bsdd-graphql-soml-orig.yaml

```diff
@@ -0,0 +1,1149 @@
+        label: The unique identification of the domain
+        range: string
+        max: 1
+        nonNullable: true
+      status:
+        name: status
+        label: ''
+        range: string
+        max: 1
+        nonNullable: false
+      releaseDate:
+        name: releaseDate
+        label: ''
+        range: dateTime
+        max: 1
+        nonNullable: false
+      license:
+        name: license
+        label: The type of license for using data of this domain
+        range: string
+        max: 1
+        nonNullable: false
+      licenseUrl:
+        name: licenseUrl
+        label: Url with more info about the license
+        range: string
+        max: 1
+        nonNullable: false
+      copyrightNotice:
+        name: copyrightNotice
+        label: The domain copyright
+        range: string
+        max: 1
+        nonNullable: false
+      moreInfoUrl:
+        name: moreInfoUrl
+        label: Url with more info about the domain
+        range: string
+        max: 1
+        nonNullable: false
+      lastUpdatedUtc:
+        name: lastUpdatedUtc
+        label: UTC date this domain was last updated
+        range: dateTime
+        max: 1
+        nonNullable: true
+      languageCode:
+        name: languageCode
+        label: The language the results will be returned in if no language specified
+          or if specified language is not available
+        range: string
+        max: 1
+        nonNullable: true
+      classificationSearch:
+        name: classificationSearch
+        label: Search for classifications in the domain. Results are limited to max
+          5000 items.
+        range: Classification
+        max: inf
+        nonNullable: false
+      classification:
+        name: classification
+        label: Select a classification
+        range: Classification
+        max: 1
+        nonNullable: false
+  Language:
+    name: Language
+    kind: object
+    label: ''
+    type: bsdd:Language
+    props:
+      isocode:
+        name: isocode
+        label: Unique identification of the language
+        range: string
+        max: 1
+        nonNullable: true
+      name:
+        name: name
+        label: Language name
+        range: string
+        max: 1
+        nonNullable: true
+  Property:
+    name: Property
+    kind: object
+    label: null
+    type: bsdd:Property
+    props:
+      allowedValues:
+        name: allowedValues
+        label: List of allowed values
+        range: PropertyValue
+        max: inf
+        nonNullable: false
+      connectedPropertyCodes:
+        name: connectedPropertyCodes
+        label: List of codes of connected properties
+        range: string
+        max: inf
+        nonNullable: false
+      dataType:
+        name: dataType
+        label: 'Format for expressing the value of the property: Boolean, Character,
+          Date, Enumeration, Integer, Real, String, Time'
+        range: string
+        max: 1
+        nonNullable: false
```

Comment:
- `nonNullable: false` means `min: 0` and is default, so you don't need to emit it.
- `range: string` is default, so you don't need to emit it

The shorter SOML you can generate, the better!

Reply:
Updated to override only when the values differ from their defaults